docs/products/flink/howto/rest-api-cheatsheet.rst

```diff
@@ -0,0 +1,73 @@
+Flink REST API examples cheatsheet
+==================================
+
+This page contains an overview of the REST API calls examples useful to create Aiven for Apache Flink table and job definitions.
+
+Create PostgreSQL based Flink table
+-------------------------------------------------
+
+Define a Flink table named ``node_info``
+
+* pointing to a PostgreSQL table named ``public.node_definition`` available in the PostgreSQL database identified by the integration with id ``a4af7409-d167-4f31-af13-ddd4cd06564f``. 
+* using the Aiven for Flink service named ``my-flink-service`` in the ``my-test-project`` project.
+
+Endpoint::
+
+    POST https://api.aiven.io/v1/project/my-test-project/test/service/my-flink-service/flink/table
+
+Header: ``Authorization Bearer TOKEN``
+
+Body::
+
+    {
+    "integration_id": "a4af7409-d167-4f31-af13-ddd4cd06564f",
+    "name": "node_info",
+    "jdbc_table": "public.node_definition",
+    "schema_sql": "`node` INT, `node_description` VARCHAR"
+    }
+
+Create Kafka based Flink table
+-------------------------------------------------
+
+Define a Flink table named ``KAlert`` 
+
+* pointing to a Kafka topic named ``alert`` available in the Kafka cluster identified by the integration with id ``a4af7409-d167-4f31-af13-ddd4cd06564f``
```

Comment:
Same as above, "id" could be capitalized here and preceded by an article -> "...integration with the ID..."

Reply:
Fixed